CONTRIBUTING.md

```diff
@@ -1,5 +1,220 @@
 # Contributing to React Native
 
-Want to contribute to React Native? There are a few things you need to know.
+<!-- generated_contributing_start -->
```

Comment:
This is autogenerated. Do not review. See changes to `docs/Contributing.md` when revewing.